biolink-model.yaml

```diff
@@ -6145,6 +6145,717 @@ slots:
+    description: >-
+      Log-Frequency-Biased Mutual Dependency is a variant of Mutual Dependency
+      that results in favoring the more frequently occuring pairs when ranking. For
+      details, see Thanopoulos, A., Fakotakis, N., & Kokkinakis, G. (2002, May). Comparative
+      Evaluation of Collocation Extraction Metrics. In LREC (Vol. 2, pp. 620-625).
+      http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.11.8101&rep=rep1&type=pdf
+    range: string
+
+  tmkp concept1 count:
+    is_a: association slot
+    description: The raw count of observations of one member (concept1) of a cooccurring
+      pair of concepts in a text corpus.
+    range: string
+
+  tmkp concept2 count:
+    is_a: association slot
+    description: The raw count of observations of one member (concept2) of a cooccurring
+      pair of concepts in a text corpus.
+    range: string
+
+  tmkp concept cooccurrence count:
+    is_a: association slot
+    description: The raw count of observations of two concepts cooccurring in the
+      same document (or part of a document) in a text corpus.
+    range: string
+
+  contribution towards patient survival:
+    is_a: association slot
+    description: >-
+      Contribution is a summary statistic indicating the level of contribution
+      an entity has on patient survival time as measured by its quantifiable impact
+      on a distribution of patient survival time.
+    range: integer
+    examples:
+      - value: '[-1,1]: e.g. -0.23, 1, 0.75'
+
+  probability of prolonged survival:
+    is_a: association slot
+    description: Reflects the posterior distribution for likliehood of survival past
+      a specified period of time, given some evidence established in the connecting
+      nodes.
+    range: integer
+    examples:
+      - value: 'e.g., .57, 1, .01'
+
+  half maximal inhibitory concentration:
+    is_a: association slot
+    description: IC50 indicates how much drug is needed to inhibit a biological process
+      by half, thus providing a measure of potency of an antagonist drug in pharmacological
+      research
+    range: datetime
+    examples:
+      - value: '-2.112848714'
+
+  binding constant:
+    is_a: association slot
+    description: measure of how likely the ligand and moelcule are to bind
+    range: integer
+    examples:
+      - value: '421'
+
+  inhibitor constant:
+    is_a: association slot
+    description: Ki, is an indication of how potent an inhibitor is; it is the concentration
+      required to produce half maximum inhibition.
+    range: integer
+    examples:
+      - value: '3000'
+
+  dissociation constant:
+    is_a: association slot
+    description: The Kd is a useful way to present the affinity of a drug for its
+      biological target. Kd tells us the concentration of drug that is required to
+      yield a significant amount of interaction with the target protein.
+    range: integer
+    examples:
+      - value: '6800'
+
+  z score:
```

Comment:
some mappings here https://bioportal.bioontology.org/search?q=Z%20score&ontologies=&include_properties=false&include_views=false&includeObsolete=false&require_definition=false&exact_match=false&categories=

you can use mark's tool to auto-infer these

I would take STATO as primary. They also have exemplar R and scipy functions which help ground these